src/ontology/mondo-edit.obo

```diff
@@ -452064,6 +452064,17 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
 relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/600 {xref="Orphanet:425"} ! APOA1
 created_by: https://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: MONDO:0100190
+name: gastric intestinal metaplasia
+def: "Metaplastic changes in the lining of the upper digestive tract." [https://my.clevelandclinic.org/health/diseases/22215-intestinal-metaplasia, PMID:29606921]
+synonym: "GIM" EXACT [PMID:29606921]
+synonym: "intestinal metaplasia" EXACT [NCIT:C8360, PMID:29606921]
+xref: NCIT:C8360 {source="MONDO:equivalentTo"}
+is_a: MONDO:0004298 {source="https://orcid.org/0000-0001-7151-1615"} ! stomach disorder
```

Comment:
Maybe there could be another parent: MONDO:0021074 - precancerous condition 
According to PMID:29606921:"Gastric intestinal metaplasia is a precancerous change of the mucosa of the stomach with intestinal epithelium, and is associated with an increased risk of dysplasia and cancer."